test/src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -321,6 +321,10 @@ public Jenkins getInstance() {
      * @throws Throwable if setup fails (which will disable {@code after}
      */
     public void before() throws Throwable {
+        if (Thread.interrupted()) { // JENKINS-30395
+            System.err.println("was interrupted before start");
```

Comment:
There is no need for the logging system from a test run; that would just wind up dumping the same message on stderr anyway.